### ISSUE TITLE
Custom user agent.

### ISSUE THREAD

**Issue by raedok** [mafintosh/peerflix#315]
Can an option be added to set custom user agents when getting peers from the tracker? Some trackers forbid usage of non main-stream torrent clients.

**Comment by mafintosh** [comment]
@feross does torrent-discovery support anything like that o/

**Comment by raedok** [comment]
@mafintosh According to https://www.npmjs.com/package/torrent-discovery there's an optional setting for user-agent.

**Comment by feross** [comment]
Yep, just set the `userAgent` option in the constructor to a string.
